modules/consensus/consensusset_bench_test.go

```diff
@@ -0,0 +1,19 @@
+package consensus
+
+import (
+	"strconv"
+	"testing"
+)
+
+// BenchmarkCreateServerTester benchmarks creating a server tester from
+// scratch. The consensus package creates over 60 server testers (and
+// counting), and optimizations to the server tester creation process are
+// likely to generalize to the project as a whole.
+func BenchmarkCreateServerTester(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		_, err := createConsensusSetTester("BenchmarkCreateServerTester - " + strconv.Itoa(i))
```

Comment:
yikes, how many folders do you wind up with? And shouldn't these be closed after being created?